trackpy/linking/linking.py

```diff
@@ -53,9 +53,11 @@ def link_iter(coords_iter, search_range, **kwargs):
         algorithm used to resolve subnetworks of nearby particles
         'auto' uses hybrid (numba+recursive) if available
         'drop' causes particles in subnetworks to go unlinked
-    dist_func : function, optional
-        a custom distance function that takes two 1D arrays of coordinates and
-        returns a float. Must be used with the 'BTree' neighbor_strategy.
+    dist_func : function or ```sklearn.metrics.DistanceMetric``` class, optional
```

Comment:
I think this is supposed to be an instance of `DistanceMetric`, not a class.

Reply:
Ah, good catch.

Reply:
My new commits should hopefully resolve my poor choice of documentation here.